### DISCUSSION TITLE
QQ适配器的NoneBot卡在了Checking for matchers completed

### DISCUSSION THREAD

**Discussion by 1292917512** [nonebot/discussions#509] [问答]
![20240912-210505](https://github.com/user-attachments/assets/2d99ef2e-4729-4e40-82d5-ee85f30fbf21)
具体表现如图所示，官方的Bot是成功连接了。到这里就一直卡着不动了。 echo指令也没有应答应该是机器人还没启动完成。

**Comment by 1292917512** [comment]
适配器为~fastapi+~httpx+~websockets

**Comment by yanyongyu** [comment]
这不是卡死，就是全部执行完毕了。。。你说的echo指令没有应答，从截图日志上看你就没有发消息给机器人，请检查你发送消息的频道是否正确。

**Reply by 1292917512** [reply]
问题找到了是官方配置没有配置功能指令。感谢大佬~ 回复的真快~

**Reply by mmjon** [reply]
大佬怎么解决的
